src/styles/utils/_mixin.scss

```
@@ -0,0 +1,5 @@
+
+@mixin square($day-size) {
```

You don't need empty line at the start of the file.
```suggestion
@mixin square($day-size) {
```